experimental/traffic-portal/src/app/core/profiles/profile-detail/profile-detail.component.ts

```diff
@@ -0,0 +1,151 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import { Component, OnInit } from "@angular/core";
+import { MatDialog } from "@angular/material/dialog";
+import { ActivatedRoute, Router } from "@angular/router";
+import { ProfileType, ResponseCDN, ResponseProfile } from "trafficops-types";
+
+import { CDNService, ProfileService } from "src/app/api";
+import { DecisionDialogComponent } from "src/app/shared/dialogs/decision-dialog/decision-dialog.component";
+import { NavigationService } from "src/app/shared/navigation/navigation.service";
+
+/**
+ * ProfileDetailComponent is the controller for the profile add/edit form.
+ */
+@Component({
+	selector: "tp-profile-detail",
+	styleUrls: ["./profile-detail.component.scss"],
+	templateUrl: "./profile-detail.component.html"
+})
+export class ProfileDetailComponent implements OnInit {
+	public new = false;
+
+	/** Loader status for the actions */
+	public loading = true;
+
+	/** All details of profile requested */
+	public profile!: ResponseProfile;
+
+	/** All cdns used for profile creation as input */
+	public cdns!: ResponseCDN[];
+
+	public types = [
```

Comment:
These values are provided by the `ProfileType` enum. Not sure if those can be directly bound to a select - if they can it should just use `ProfileType` then. If not, these values should be `ProfileType.ATS_PROFILE` instead of 'hardcoded' strings.